util/packaging/homebrew/chapel-release.rb

```diff
@@ -1,34 +1,50 @@
 class Chapel < Formula
   desc "Programming language for productive parallel computing at scale"
   homepage "https://chapel-lang.org/"
-  url "https://github.com/chapel-lang/chapel/releases/download/1.26/chapel-1.26.0.tar.gz"
-  sha256 "ba396b581f0a17f8da3f365a3f8b079b8d2e229a393fbd1756966b0019931ece"
-#  url "<URL of Chapel tarball>"
-#  sha256 "<sha256 sum for the associated tarball>"
+  url "https://github.com/chapel-lang/chapel/releases/download/1.27.0/chapel-1.27.0.tar.gz"
+  sha256 "558b1376fb7757a5e1f254c717953f598a3e89850c8edd1936b8d09c464f3e8b"
   license "Apache-2.0"
 
   bottle do
-    sha256 arm64_monterey: "229f22e29b0cc7a904841636c924d0c94c00299f817ed294247957d43c128cf3"
-    sha256 arm64_big_sur:  "9f4a5b7644f18da492f75b4442bc7e22d0b59fa72d6952b15f14f15a2949f57b"
-    sha256 monterey:       "bd3d8c066ce51d66b44e0430fc12f01ce5b99701f337304bf609b185bdbdefec"
-    sha256 big_sur:        "20b698193b5f7efb99c0eee60126a3e47a420fe99a65ed83aba919fac1cfacac"
-    sha256 catalina:       "8eeb73c3884680831146792096af481b1f2c23902eed1ad798f3aa890d2a1e49"
-    sha256 x86_64_linux:   "f351e793925739313b0025b6e90287688da278ebe60191000b5a6e5e239b60ef"
+    sha256 arm64_monterey: "2f3b638cc2c187ac2c11f0dc128eae825466ce6745b5ad358e2fee8b96ef0187"
+    sha256 arm64_big_sur:  "90f68a1e21e3ad61165bbcea1e79306cbf09e811f5b5dd0c91068dadb07e0aca"
+    sha256 monterey:       "f33951b329c819f7ff386b33a413b3cd331fb3a31c0823f10f435d0227c29cf2"
+    sha256 big_sur:        "aeaa92c2f32435c1c16c52f737261c66b2b44ce197ec3e6f6a479f1c029d702c"
+    sha256 catalina:       "11eebc6dbc8383482c689e8a1d74359d12a4781579a48bb29417ca25779c50c4"
+    sha256 x86_64_linux:   "c4ccfadfa28118b70d76a324310dc17be9f351011490c7c29eb6dca5eb7ab093"
   end
 
   depends_on "gmp"
-  depends_on "python@3.10"
+  # `chapel` scripts use python on PATH (e.g. checking `command -v python3`),
+  # so it needs to depend on the currently linked Homebrew Python version.
+  # TODO: remove from versioned_dependencies_conflicts_allowlist when
+  # when Python dependency matches LLVM's Python for all OS versions.
+  depends_on "python@3.9"
+
   on_macos do
     depends_on "llvm" if MacOS.version > :catalina
+    # fatal error: cannot open file './sys_basic.h': No such file or directory
+    # Issue ref: https://github.com/Homebrew/homebrew-core/issues/96915
     depends_on "llvm@11" if MacOS.version <= :catalina
   end
+
   on_linux do
     depends_on "llvm"
   end
 
   # LLVM is built with gcc11 and we will fail on linux with gcc version 5.xx
   fails_with gcc: "5"
 
+  # Work around Homebrew 11-arm64 CI issue, which outputs unwanted objc warnings like:
+  # objc[42134]: Class ... is implemented in both ... One of the two will be used. Which one is undefined.
+  # These end up incorrectly failing checkChplInstall test script when it checks for stdout/stderr.
+  # TODO: remove when Homebrew CI no longer outputs these warnings or 11-arm64 is no longer used.
+  patch :DATA
```

Comment:
Shouldn't this patch be in the chapel-main.rb version as well? Since it is about Homebrew CI & the fix isn't something we have (or expect to have) in chapel's `main` ?

Reply:
The patch is part of chapel-main.rb.  It was added when the Homebrew team updated our formula for llvm14.